docs/rollen/index.md

```diff
@@ -1,29 +1,40 @@
 ---
 title: Rollen
+summary: Hulp bij het bepalen van rollen en verantwoordelijkheden om als overheid verantwoordelijk om te gaan met algoritmes en AI.
 icon: material/account
 ---
+# Rollen
+Verantwoorde algoritmes en AI-systemen zijn mogelijk als mensen in verschillende rollen goed samenwerken, op het juiste moment.
 
-In dit deel van het Algoritmekader kan per rol informatie worden geraadpleegd over het verantwoord ontwikkelen en gebruiken van algoritmes en AI. 
-Om verantwoorde inzet van algoritmes en AI te bereiken is een samenspel nodig tussen verschillende expertises die op het juiste moment worden ingezet. 
-In de praktijk creëren organisaties specifieke rollen of functies (hierna: rol) waarbij wordt verwacht dat medewerkers met deze rol de gevraagde expertise toepassen. 
-Zo zal een data scientist betrokken zijn bij het ontwikkelen en trainen van algoritme en AI en zal een privacy officer maatregelen definiëren voor het beschermen van persoonsgegevens. 
+## Bepaal zelf rollen en verantwoordelijkheden
+Organisaties bepalen zelf de namen van hun rollen of functies en de taken en verantwoordelijkheden die hierbij horen. 
+Sommige organisaties hebben bijvoorbeeld minder rollen, omdat zij verschillende expertises combineren in één rol. Of rollen hebben andere namen.
 
-In het Algoritmekader wordt een poging gedaan om te duiden welke rol(len) bij de realisatie van vereisten en maatregelen betrokken (kunnen) zijn. 
-Deze laag van het Algoritmekader dient om meerdere redenen alleen ter inspiratie. 
-Dat komt bijvoorbeeld omdat organisaties andere rollen gebruiken, bepaalde expertise samenvoegen in één rol of omdat bepaalde expertise nog niet is verworven binnen een organisatie. 
-Het koppelen van vereisten en maatregelen aan rollen raakt ook het onderwerpen van taakstelling en verantwoordelijkheden. 
-Ook dit zal bij organisaties op verschillende manieren belegd zijn of worden. 
+## Voorbeelden van rollen
```

Comment:
Ik vraag me af of je hier ook een opsomming wilt? Er is ook al een menu aan de linkerkant van de pagina?

Reply:
Graag linkermenu weghalen.

Reply:
voor nu gekozen om het linkermenu voor nu te laten staan. Dat wordt automatisch gemaakt. Gezien de rollen nog veranderlijk zijn, heeft dat de voorkeur. Een handmatige lijst zorgt ervoor dat rollen op twee plekken moeten worden toegevoegd. Dat is foutgevoelig. Wanneer we overgaan tot een andere huisstijl kan dit mogelijk op een andere manier opgelost worden.